specification/search/resource-manager/Microsoft.Search/preview/2024-03-01-preview/examples/SearchListOperations.json

```diff
@@ -0,0 +1,616 @@
+          },
+          {
+            "name": "Microsoft.Search/searchServices/listAdminKeys/action",
+            "isDataAction": false,
+            "display": {
+              "provider": "Microsoft Search",
+              "resource": "Search Services",
+              "operation": "Get Admin Key",
+              "description": "Reads the admin keys."
+            },
+            "origin": "user,system"
+          },
+          {
+            "name": "Microsoft.Search/searchServices/regenerateAdminKey/action",
+            "isDataAction": false,
+            "display": {
+              "provider": "Microsoft Search",
+              "resource": "Search Services",
+              "operation": "Regenerate Admin Key",
+              "description": "Regenerates the admin key."
+            },
+            "origin": "user,system"
+          },
+          {
+            "name": "Microsoft.Search/searchServices/listQueryKeys/action",
+            "isDataAction": false,
+            "display": {
+              "provider": "Microsoft Search",
+              "resource": "API Keys",
+              "operation": "Get Query Keys",
+              "description": "Returns the list of query API keys for the given Azure AI Search service."
+            },
+            "origin": "user,system"
+          },
+          {
+            "name": "Microsoft.Search/searchServices/createQueryKey/action",
+            "isDataAction": false,
+            "display": {
+              "provider": "Microsoft Search",
+              "resource": "Search Services",
+              "operation": "Create Query Key",
+              "description": "Creates the query key."
+            },
+            "origin": "user,system"
+          },
+          {
+            "name": "Microsoft.Search/searchServices/deleteQueryKey/delete",
+            "isDataAction": false,
+            "display": {
+              "provider": "Microsoft Search",
+              "resource": "API Keys",
+              "operation": "Delete Query Key",
+              "description": "Deletes the query key."
+            },
+            "origin": "user,system"
+          },
+          {
+            "name": "Microsoft.Search/checkNameAvailability/action",
+            "isDataAction": false,
+            "display": {
+              "provider": "Microsoft Search",
+              "resource": "Service Name Availability",
+              "operation": "Check Service Name Availability",
+              "description": "Checks availability of the service name."
+            },
+            "origin": "user,system"
+          },
+          {
+            "name": "Microsoft.Search/searchServices/diagnosticSettings/read",
+            "isDataAction": false,
+            "display": {
+              "provider": "Microsoft Search",
+              "resource": "Diagnostic Settings",
+              "operation": "Get Diagnostic Setting",
+              "description": "Gets the diganostic setting read for the resource"
```

Comment:
some of these descriptions don't have periods.  Can you do a quick scan?  this is the first one I noticed but I see the next one is also missing a period.  Also, I don't know what "diagnostic setting read" means?  Is it supposed to be read (past tense)?  It might be clearer if you use read as a verb "Reads the diagnostic settings for the search resource." or maybe just "Gets the diagnostic settings for the resource."